test/Esprima.Tests/ConverterTests.cs

```diff
@@ -12,14 +12,21 @@ public class ConverterTests
 {
     private static Module ParseExpression(string code, bool jsx = false)
     {
-        return new JavaScriptParser(code, new ParserOptions(){ Jsx = jsx }).ParseModule();
+        return new JavaScriptParser(code, new ParserOptions() { Jsx = jsx }).ParseModule();
     }
 
     private static T? FindNearTypeOfDescend<T>(Node node) where T : Node
     {
         return node.DescendantNodesAndSelf().OfType<T>().FirstOrDefault(_ => true, null);
     }
 
+    private static object? FindNearTypeOfDescendTyped(Type type, Node node)
+    {
+        return typeof(ConverterTests)
```

Comment:
Don't use reflection here. Check the type in linq directly. The generic method can then call this one.